build.sbt

```diff
@@ -1,156 +1,50 @@
-    "-Xlint:stars-align",            // Pattern sequence wildcard must align with sequence component.
-    "-Xlint:type-parameter-shadow",  // A local type parameter shadows a type already in scope.
-    "-Ywarn-dead-code",              // Warn when dead code is identified.
-    "-Ywarn-numeric-widen",          // Warn when numerics are widened.
-    "-Ywarn-value-discard",          // Warn when non-Unit expression results are unused.
-    // Scala 2.12 below
-    "-Xlint:constant",         // Evaluation of a constant arithmetic expression results in an error.
-    "-Ywarn-extra-implicit",   // Warn when more than one implicit parameter section is defined.
-    "-Ywarn-unused:implicits", // Warn if an implicit parameter is unused.
-    "-Ywarn-unused:imports",   // Warn if an import selector is not referenced.
-    "-Ywarn-unused:locals",    // Warn if a local definition is unused.
-    "-Ywarn-unused:params",    // Warn if a value parameter is unused.
-    "-Ywarn-unused:patvars",   // Warn if a variable bound in a pattern is unused.
-    "-Ywarn-unused:privates",  // Warn if a private member is unused.
-
-    // Inlining
-    "-opt:l:inline",
-    "-opt:l:method",
-    "-opt-inline-from:com.permutive.**",
-    "-opt-warnings",
-    // Lint after expansion so that implicits used in macros are not flagged as unused
-    "-Ywarn-macros:after",
-  ),
-  scalacOptions ++= {
-    if (priorTo2_13(scalaVersion.value))
-      Seq(
-        "-Yno-adapted-args",                // Do not adapt an argument list (either by inserting () or creating a tuple) to match the receiver.
-        "-Ypartial-unification",            // Enable partial unification in type constructor inference
-        "-Ywarn-inaccessible",              // Warn about inaccessible types in method signatures.
-        "-Ywarn-infer-any",                 // Warn when a type argument is inferred to be `Any`.
-        "-Ywarn-nullary-override",          // Warn when non-nullary `def f()' overrides nullary `def f'.
-        "-Xlint:nullary-override",          // Warn when non-nullary `def f()' overrides nullary `def f'.
-        "-Ywarn-nullary-unit",              // Warn when nullary methods return Unit.
-        "-Xlint:by-name-right-associative", // By-name parameter of right associative operator.
-        "-Xlint:unsound-match",             // Pattern match may not be typesafe.
-        "-Xfuture",                         // Turn on future language features.
-      )
-    else
-      Seq(
-        "-Ymacro-annotations",
-      )
-  }
+ThisBuild / organization := "com.permutive"
+ThisBuild / organizationName := "Permutive"
+ThisBuild / organizationHomepage := Some(url("https://github.com/permutive"))
+ThisBuild / licenses := Seq(License.Apache2)
+ThisBuild / developers := List(
+  tlGitHubDev("cremboc", "Paulius Imbrasas"),
+  tlGitHubDev("TimWSpence", "Tim Spence"),
+  tlGitHubDev("bastewart", "Ben Stewart"),
+  tlGitHubDev("travisbrown", "Travis Brown")
```

Comment:
Do we need to remove old developers from this list or do we keep them here?

Reply:
Good question! I'm not sure...